### PR TITLE
CODE RUB: Syntax corrections

### PR DESCRIPTION
**Pull request by ziyadullayevmirjalol**

### CODE REVIEW

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.cs [hunk 1]
```diff
@@ -2,84 +2,82 @@
...
 using EFxceptions;
+using GitFyle.Core.Api.Models.Foundations.Contributions;
 using GitFyle.Core.Api.Models.Foundations.ContributionTypes;
 using GitFyle.Core.Api.Models.Foundations.Contributors;
 using GitFyle.Core.Api.Models.Foundations.Repositories;
 using GitFyle.Core.Api.Models.Foundations.Sources;
-using GitFyle.Core.Api.Models.Foundations.Contributions;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
+using System.Linq;
```
[the remark this answers is not in the document]
Comment (ziyadullayevmirjalol) [reply GitFyle.Core.Api/Brokers/Storages/StorageBroker.cs]: done

**Comment by glhays** [comment]
@ziyadullayevmirjalol what makes your syntax override default VS syntax. Your syntax corrections will force every user implementing a default VS class adjust to your preferential syntax. Please explain??

**Comment by ziyadullayevmirjalol** [comment]
@glhays Hello, I wanted to make the code more readable because using semicolons after namespaces will make the code easier to read and escapes another extra curly brackets that is in the namespace and I think if we want to increase the readability of our code we should use this way, and there are also too many ways to make the code readable. If this way is not correct and harder to use for people, Ok we don't use this.

**Comment by cjdutoit** [comment]
@ziyadullayevmirjalol thanks for the effort.  We have not adopted the shorthand introduced in C# 10.0 known as file-scoped namespace declarations. I will close this PR.  Please check the open issues for other opportunities to contribute.

**Comment by ziyadullayevmirjalol** [comment]
> @ziyadullayevmirjalol thanks for the effort. We have not adopted the shorthand introduced in C# 10.0 known as file-scoped namespace declarations. I will close this PR. Please check the open issues for other opportunities to contribute.

@cjdutoit thank you, I will

**Pull request closed without merging**